### ISSUE TITLE
Update Integrating Loss Functions topic to include TensorFlow

### ISSUE THREAD

**Issue by jlema** [microsoft/BackwardCompatibilityML#111]
This section needs  to be updated to include guidance for TensorFlow.
https://backwardcompatibilityml.readthedocs.io/en/latest/backwardcompatibilityml/topics/integrating_loss_functions.html
